site/content/en/docs/Installation/Creating Cluster/gke.md

```diff
@@ -173,12 +186,12 @@ recommended for a production deployment.
 ```bash
 gcloud container node-pools create agones-system \
   --cluster=[CLUSTER_NAME] \
-  --no-enable-autoupgrade \
+  --region=[COMPUTE_REGION] \
   --node-taints agones.dev/agones-system=true:NoExecute \
   --node-labels agones.dev/agones-system=true \
   --num-nodes=1
 ```
-where [CLUSTER_NAME] is the name of the cluster you created.
+where `[CLUSTER_NAME]`is the name of the cluster you created and `[COMPUTE_REGION]` is the region.
```

Comment:
This section is missing flag explanations. It didn't seem strange until I saw that both the previous and next sections have them, so they should be added here too.

Reply:
Actually, I think they should be moved here and removed below. Looking at how they're explained, below we only explain the _new_ flags.